app/src/main/java/org/wikipedia/staticdata/MainPageNameData.java

```diff
@@ -5,6 +5,8 @@
 
```

Comment:
This is an auto-generated file, and will be overwritten the next time we update our languages.  You probably intended to edit the `scripts/templates/basichash.java.jinja` file.

Reply:
Thanks! You are correct. Done!